multicluster/test/e2e/service_test.go

```diff
@@ -27,75 +27,108 @@ import (
 	e2euttils "antrea.io/antrea/test/e2e/utils"
 )
 
-func testServiceExport(t *testing.T, data *MCTestData) {
-	data.testServiceExport(t)
+func TestMCService(t *testing.T) {
+	data, err := setupTest(t)
+	defer teardownTest(t, data)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+
+	podName := data.setupTestResources(t)
+	defer data.tearDownTestResources(t, podName)
+
+	t.Run("TestMCServiceExport", func(t *testing.T) {
+		setUpServiceExport(data, t)
+		t.Run("Case=Connectivity", func(t *testing.T) { testProbeMCService(t, data) })
+		t.Run("Case=ANPNoPriority", func(t *testing.T) { testANP(t, data) })
```

Comment:
nit: maybe rename this testcase to "ANPDropToMCService"

Reply:
Sure, on it. Thanks!